cognite/neat/_rules/importers/_rdf/_shared.py

```diff
@@ -401,25 +376,12 @@ def fix_description(metadata: dict, default: str = "This model has been inferred
     return metadata
 
 
-def fix_prefix(metadata: dict, default: str = "neat") -> dict:
-    if prefix := metadata.get("prefix", None):
-        if not isinstance(prefix, str) or not PATTERNS.prefix_compliance.match(prefix):
-            metadata["prefix"] = default
-    else:
-        metadata["prefix"] = default
-    return metadata
-
-
-def fix_namespace(metadata: dict, default: Namespace) -> dict:
-    if namespace := metadata.get("namespace", None):
-        if not isinstance(namespace, Namespace):
-            try:
-                metadata["namespace"] = Namespace(namespace)
-            except Exception:
-                metadata["namespace"] = default
+def fix_space(metadata: dict, default: str = "neat") -> dict:
+    if space := metadata.get("space", None):
+        if not isinstance(space, str) or not PATTERNS.space_compliance.match(space):
```

Comment:
A consequence now is that Information rules needs to use a prefix that is compliant with the regex for CDF space?

Reply:
Yes, we are forcing users to closed world